package com.docker.yyf.exception;

public class AlreadyLockedException extends RuntimeException {
}
